src/common/http.js

```diff
@@ -112,12 +112,16 @@ axios.interceptors.response.use(
           Array.isArray(error.config.errorsHandleCustom) &&
           error.config.errorsHandleCustom.includes(status)
         if (!handleErrorSelf) {
```

Comment:
This file is an original JavaScript file, so I temporarily removed the TypeScript syntax, util we move it to the ts file.

Reply:
🧐